axelor-account/src/main/resources/views/BudgetDistribution.xml

```diff
@@ -14,5 +14,37 @@
        		<field name="amount"/>
        </panel>
     </form>
+
+    <grid name="budget-distribution-grid-purchase-order-line-grid" title="Committed lines" model="com.axelor.apps.account.db.BudgetDistribution">
+        <field name="purchaseOrderLine.purchaseOrder"/>
+        <field name="purchaseOrderLine.purchaseOrder.orderDate"/>
+        <field name="purchaseOrderLine.product"/>
+        <field name="amount"/>
+    </grid>
+
+    <form name="budget-distribution-grid-purchase-order-line-form" title="Committed line" model="com.axelor.apps.account.db.BudgetDistribution" width="large">
+        <panel name="mainPanel" >
```

Comment:
Is it possible to simplify this code by putting readonly="true" in form or panel tag ?